Packs/Exabeam/Integrations/Exabeam/Exabeam.py

```diff
@@ -103,7 +103,7 @@ def test_module_request(self):
         """
         self._http_request('GET', full_url=f'{self._base_url}/api/auth/check', resp_type='text')
 
-    def get_notable_users_request(self, api_unit: str = None, num: str = None, limit: int = None) -> Dict:
+    def get_notable_users_request(self, api_unit: str = None, num: str = None, limit: int = None) -> dict:
```

Comment:
Did you change it or the pre-commit hook?

Reply:
pre-commit

Reply:
pre-commit